online-banking-app/pom.xml

```diff
@@ -61,6 +61,20 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.1.0</version>
```

Comment:
Please send version number to top most parent pom. And declare dependency in the dependency management of that pom.